src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -38,34 +39,38 @@ const ChannelList = () => {
   return (
     <>
       <Drawer anchor="left" open={drawer} onClose={() => setDrawer(false)}>
-        <Box sx={{ width: 250 }} role="presentation" className={styles.upperGroup}>
+        <Box sx={{ width: 300 }} role="presentation" className={styles.upperGroup}>
```

Comment:
Leave a 'TODO' comment here so we know we have to change this value later on... Fixed values are no good for responsive design

Reply:
OK